### PR TITLE
1 as a user i want to create a list of accountids to add to my allowlist

### LINKED ISSUE #1: As a User, I want to create a list of accountIds to add to my allowlist
Acceptance criteria:
- ~~This should use the snapshot tool to get the snapshot from the mirror node~~
- The app should take a tokenId and threshold for the amount of owned tokens within given tokenId
- The app should return an array of accountIds that fulfill the criteria as output
- Query should take all the pages into account (if links.next is not null, iterate through)
- The app is hosted on GitHub pages

Example use case:
- find all accounts that have at least 3 NFTs from X collection

example mirror node query:
https://mainnet-public.mirrornode.hedera.com/api/v1/tokens/0.0.3103996/balances?account.balance=gte:2&limit=100

Image

### PR DESCRIPTION
**Pull request by StanislawMazowieckiAriane**
Fixes:
#1 
#2 
#3 
#4

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  }
+}
+
+.card {
+  padding: 2em;
+}
+
+.read-the-docs {
+  color: #888;
+}
```
Comment (michalrozekariane) [inline src/App.css]: I think that styles you need from `App.css` can be moved to `index.css` and after that you can delete this file

File: src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+  useEffect(() => {
+    if (isSuccess) toast.success(dictionary.successfullyFetchedData);
+  }, [isSuccess]);
+
+  useEffect(() => {
+    if (error) {
+      toast.error(error.toString());
+    }
+  }, [error]);
```
Comment (michalrozekariane) [inline src/App.tsx:82]: You can probably move toast logic into try / catch block inside `fetchData` func

Comment (StanislawMazowieckiAriane) [reply src/App.tsx:82]: I preffer to throw all errors in fetchData and use error from useQuery to display it. I think it's more efficient way

File: src/App.css (hunk 1 above)
Comment (StanislawMazowieckiAriane) [reply src/App.css]: Done

**Review by michalrozekariane — APPROVED** [review]

**Review by BartoszSolkaArianelabs — APPROVED** [review]

**Pull request merged**